secure_message/repository/modifier.py

```diff
@@ -15,10 +15,26 @@
 class Modifier:
     """Modifies message to add / remove statuses"""
 
+
+    @staticmethod
+    def _get_label_actor(user, message):
+        try:
+            if user.is_respondent:
+                actor = user.user_uuid
+            elif message['from_internal']:
+                actor = message['msg_from']
+            else:
+                actor = message['msg_to'][0]
+        except KeyError:
+            logger.exception("Failed to remove label, no msg_to field")
+            raise InternalServerError(description="Error getting actor details from message")
+
+        return actor
+
     @staticmethod
     def add_label(label, message, user, session=db.session):
         """add a label to status table"""
-        actor = user.user_uuid if user.is_respondent else constants.BRES_USER
```

Comment:
I think we needed to keep the constant.BRES_USER because of something to do with the old R-OPS using it? I can't remember because Torrance was working on it, but definitely something we need to check.

Reply:
Discussed and should be okay, waiting for unit tests 👍